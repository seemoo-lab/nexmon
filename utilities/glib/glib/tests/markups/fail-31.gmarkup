<foo>&#34592348345343453453455645765736575865767;</foo>