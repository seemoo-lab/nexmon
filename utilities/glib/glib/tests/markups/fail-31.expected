ELEMENT 'foo'
ERROR Error on line 1: Failed to parse '34592348345343453453455645765736575865767', which should have been a digit inside a character reference (&#234; for example) - perhaps the digit is too large
